packages/tools/src/enums/ChangeTypes.ts

```diff
@@ -35,6 +35,10 @@ enum ChangeTypes {
    * Occurs when an annotation is changed do to an undo or redo.
    */
   History = 'History',
+  /**
+   * Send when the metadata selector changes
```

Comment:
is this about a changeType for annotation? can we get more docs please, Occurs when annotation's metadata changs? what is a metadata selector?